Neos.Neos/Resources/Private/Fusion/Prototypes/NodeLink.fusion

```diff
@@ -0,0 +1,34 @@
+# Renders an <a> tag based on Neos.Neos:NodeUri pointing to a node
+#
+prototype(Neos.Neos:NodeLink) < prototype(Neos.Fusion:Tag) {
+  node = null
+  additionalParams = Neos.Fusion:RawArray
+  arguments = Neos.Fusion:RawArray
+  argumentsToBeExcludedFromQueryString = Neos.Fusion:RawArray
+  addQueryString = false
+  absolute = false
+  baseNodeName = 'documentNode'
+
+  @context.node = ${this.node}
+  @context.additionalParams = ${this.additionalParams}
+  @context.arguments = ${this.arguments}
+  @context.argumentsToBeExcludedFromQueryString = ${this.argumentsToBeExcludedFromQueryString}
+  @context.addQueryString = ${this.addQueryString}
+  @context.absolute = ${this.absolute}
+  @context.baseNodeName = ${this.baseNodeName}
```

Comment:
nitpick: Readability could be improved by putting this into one `@context {...}` block IMO